modules/microshift-security-context-constraints-overview.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// * microshift_storage/understanding-persistent-storage-microshift.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id=microshift-security-context-constraints-overview_{context}]
+= security context constraints overview
+
+You can use security context constraints (SCCs) to control permissions for the pods in your cluster. These permissions determine the actions that a pod can perform and what resources it can access. You can use SCCs to define a set of conditions that a pod must run with to be accepted into the system.
+
+[NOTE]
+====
+Only RWO volume mounts are supported. SCC could be blocked if pods are not operating with the scc contexts.
```

Comment:
small complaint. RWO is defined below this Info Note. Not sure if that can be corrected.

Reply:
Perhaps then we just need to bump this line below the link.

Also, quick nit:
```suggestion
Only RWO volume mounts are supported. SCC could be blocked if pods are not operating with the SCC contexts.
```

Reply:
done. position of note is changed and suggest change is made.